### PR TITLE
Ra 89 - fixes

### PR DESCRIPTION
**Pull request by aameijide**
* Fix tests
* Implement Dispose for IView implementors
* Update Xamarin scripts

### CODE REVIEW

File: LongoMatch.GUI/Gui/Panel/NewProjectPanel.cs [hunk 1]
```diff
@@ -136,6 +141,8 @@ public void SetViewModel (object viewModel)
 
 		protected override void OnDestroyed ()
```
Comment (ylatuya) [inline LongoMatch.GUI/Gui/Panel/NewProjectPanel.cs]: Make sure you follow the coding style, destructors-style functions should be grouped together after the constructor

File: Tests/Integration/TestDatabaseMigrationV0.cs [hunk 2]
```diff
@@ -99,6 +101,9 @@ public void TestNoOldDatabaseToMigrate ()
 			Directory.CreateDirectory (tmpPath);
 			Directory.CreateDirectory (homePath);
 
+			App.Current.homeDirectory = homePath;
+			App.Current.configDirectory = App.Current.homeDirectory;
+
 			Environment.SetEnvironmentVariable ("LONGOMATCH_HOME", tmpPath);
 			Environment.SetEnvironmentVariable ("LGM_UNINSTALLED", "1");
```
Comment (ylatuya) [inline Tests/Integration/TestDatabaseMigrationV0.cs]: Shouldn't you just call App.Init () here?

Comment (aameijide) [inline Tests/Integration/TestDatabaseMigrationV0.cs]: Yes, I will change it.

**Comment by aameijide** [comment]
Added a commit, now other test also uses VAS.Init() instead of initialize its properties.

**Pull request merged**